data/remnant missions.txt

```diff
@@ -493,5 +597,772 @@ mission "Remnant: Technology Available"
+			`	"Well met, Newcomer among the Remnant." trills the Remnant. "Fresh blood amongst us sets songs in motion that have long been in Caesura. I have a few things to finish up right now, but I hope to work with you soon."`
+			`	She gives you a polite nod and heads back the way she came.`
+			
+mission "Remnant: Return the Samples"
+	name "Return the Samples"
+	description "A researcher on <planet>, a Remnant world, has asked you to return some samples to Nasqueron."
+	source Aventine
+	stopover Nasqueron
+	to offer
+		has "license: Remnant"
+		has "event: remnant: void sprite research"
+		random < 75
+	on offer
+		conversation
+			`As you are walking through the spaceport, the researcher you previously assisted with the studies on the Void Sprites approaches you in a hustle. "We have made some new discoveries!" he exclaims in a trill. "Some of the samples that we took all those years ago are actually eggs of some kind. So the Archon must view our previous research as  being disruptive to the Void Sprite's life-cycles."`
+			`	"We have been storing these eggs in a cryogenic stasis tank, so they should still be viable and intact. Well, most of them, anyway." His notes trail off on a mournful tone.`
+			`	"Could you return some of them to Nasqueron? According to our logs, they were found floating in the lower levels of the atmosphere, where their natural buoyancy should keep them afloat."`
+			choice
+				`	"Sure, I would be glad to return them."`
+				`	"Sorry, I'm really not interested in braving the Archon again."`
+					decline
+
+			`	"Thank you. Since we are continuing to work together, I'll try again." He takes a breath, concentrates, and speaks a short verse about plumage with a complex chord that sounds at once grandiose and self-depreciating. The best you can make out is "Plume." "We can't pay you for this, but if it does appease the Archon we will be able to resume our studies, and who knows where that could lead?"`
+				accept
+	npc
+		government " Drak "
+		system Nenia
+		personality staying heroic nemesis uninterested
+		ship "Archon (Cloaked)" "Lifted Lorax"
+	on stopover
+		conversation
+			`Once again you have successfully dodged the Archon to land on Nasqueron. Somehow, knowing it was going to be there was both better and worse than the surprise of the first time.`
+			`	As <ship> sinks through the cloud layers, you gently activate the repulsors to slow your descent to avoid startling the Void Sprites. After what seems like an eternity, you notice a few nearby sprites descending as well, and you match speeds with them, slowing as you reach the altitude listed in the log. `
```

Comment:
`<first> <last>` is a starship captain, so I assume they are probably well versed in basic physics, so specifying velocity instead of speed is probably a good change. Done.